shared/middleware/net_sim_api.rb

```diff
@@ -79,7 +66,6 @@ def has_json_utf8_headers(request)
     table = get_table(shard_id, table_name)
     int_id = id.to_i
     table.delete(int_id)
```

Comment:
Thinking out loud...

When we merge, this is where we do the 'cascading delete' cleanup work.  We had a minor optimization to send only one invalidation per table when deleting multiple wire/message rows as part of cleanup.  With the new interface we'll get an invalidation per row... until we get to the next step of having something like a `delete_if` operation that could send a single invalidation when it's done.